packages/docs/src/css/overrides.css

```diff
@@ -97,11 +98,25 @@ table thead tr {
     width: 20px;
   }
 }
-[class*="navbarLanding"] .header-github:before,
-[class*="navbarLanding"] .header-github:hover:before {
+.header-github:before,
+.header-github:hover:before {
   background-color: var(--swm-navy-light-100);
 }
 
 [class*="navbar__item"] {
   display: flex !important;
 }
+
+/* TODO: probably to remove */
+[class*="logoWrapper"] [class*="sidebarLogo"] {
+  margin-left: 1.5rem;
+}
```

Comment:
If it is _"probably to remove"_, why not just remove it?

Reply:
good thinking, removed 👍